src/Tribe/Views/V2/iCalendar/Links/Outlook_Export.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Handles Outlook iCalendar export links.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\iCalendar
+ */
+
+namespace Tribe\Events\Views\V2\iCalendar\Links;
+use Tribe\Events\Views\V2\View;
+use Tribe__Events__Main;
+
+/**
+ * Class Outlook
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\iCalendar
+ */
+class Outlook_Export extends Link_Abstract {
+	/**
+	 * {@inheritDoc}
+	 */
+	public static $slug = 'outlook-ics';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function register() {
+		$this->label = __( 'Export Outlook .ics file', 'the-events-calendar' );
+		$this->single_label = $this->label;
+
+		add_filter( 'tec_views_v2_subscribe_link_outlook_ics_visibility', [ $this, 'filter_tec_views_v2_subscribe_link_outlook_ics_visibility'], 10, 2 );
```

Comment:
This isn't going to work - the slug is `outlook-ics` so this would have to be `tec_views_v2_subscribe_link_outlook-ics_visibility` (default to "doesn't show on single events")

Reply:
Ok good, because for now it will be hidden on single events.